feat: automatically retrieve all needed personal data in correct format (#141)
CombiRWTH/StaffScheduling: cases/case_catalog.md

- number of employees: 371
- ...

## Case 3: - Real Data

> Within this case the automatically extracted data from the database is stored. This excludes the `wishes_and_blocked.json` file as this includes all the manually submitted wishes of employees as well as special circumstances such as health or family-related restrictions. All different files are listed below with the corresponding description of each field within them.

Reviewer: ```suggestion
In this case, the automatically extracted data from the database is stored. The `wishes_and_blocked.json` file is an exception because it includes all the manually submitted wishes of employees, as well as special circumstances, such as health- or family-related restrictions. The different files and their corresponding descriptions are listed below.
```

## 📁 File: `employees.json`

### 📝 Description

This file contains the list of all employees within our PE (Planungseinheit), including their internal ID, personnel number, name, and their job title.

### 📐 Structure

```jsonc
{
  "employees": [             // List of all employees
    {
      "PersNr": "string",    // Personnel number as a string
      "Prim": "int",         // Internal primary key ID
      "firstname": "string", // First name of the employee
      "name": "string",      // Last name of the employee

>       "type": "string"       // Job title (may include an intern classification code)

Reviewer 2: Yes, PersNr is not necessary anymore

>       "type": "string"       // Job title (may include an intern classification code)

Author: Prim is also already changed to key, I just forgot to change it in this to later be revised document
